### DISCUSSION TITLE
Get only number of total items from a collection

### DISCUSSION THREAD

**Discussion by MuslemRahimi** [pocketbase/pocketbase#3994] [Q&A] [answered]
The title already says everything.

Is there a functionality I am not aware of how to do

```
await pb.collection("users").getFullList()
```
but only get the total number of users without their information

**Comment by ganigeorgiev** [comment, marked as answer]
There is no dedicated endpoint for this.

You can either retrieve a list with only 1 paginated item:
```js
const { totalItems } = await pb.collection("users").getList(1, 1);
```

Or if you need a more complex aggregation, you can create a helper [View collection](https://pocketbase.io/docs/collections/#view-collection) with your specific count query.

**Reply by MuslemRahimi** [reply]
is that also possible for getFullList?

**Reply by ganigeorgiev** [reply]
I don't understand your question.
`getFullList` sends `getList` requests on batches until all items are fetched.

**Reply by MuslemRahimi** [reply]
ah sorry I misunderstood the concept. Now i get it. Thanks
